example/package.json

```diff
@@ -44,6 +44,7 @@
     "@mantine/notifications": "^7.11.0",
     "@tabler/icons-react": "^3.4.0",
     "cookie": "^0.6.0",
+    "django-ai-assistant-client": "^0.0.1",
```

Comment:
It now uses the released distribution and not the local code.

Reply:
Awesome! Maybe use exact version here instead of ^?